test/floki_test.exs

```diff
@@ -451,6 +451,20 @@ defmodule FlokiTest do
     assert Floki.find("foobar<a", "a") == []
   end
 
+  # Floki.find/2 - Raw selector structs
+
+  test "find single selector structs" do
+    selector_struct = Floki.SelectorTokenizer.tokenize("a") |> Floki.SelectorParser.parse
```

Comment:
@hansihe Instead of use the parser, it's more clear to use the struct itself (`Floki.Selector`) here.